cardano/README.md

```diff
@@ -0,0 +1,69 @@
+# Cardano
+
+[![Build Status](https://travis-ci.org/input-output-hk/rust-cardano.svg?branch=master)](https://travis-ci.org/input-output-hk/rust-cardano)
+![MIT or APACHE-2 licensed](https://img.shields.io/badge/licensed-MIT%20or%20APACHE--2-blue.svg)
+![Cardano Mainnet](https://img.shields.io/badge/Cardano%20Ada-mainnet-brightgreen.svg)
+![Cardano Staging](https://img.shields.io/badge/Cardano%20Ada-staging-brightgreen.svg)
+![Cardano Testnet](https://img.shields.io/badge/Cardano%20Ada-testnet-orange.svg)
+
+Crate with all the primitives for the [Cardano](https://www.cardano.org) blockchain:
+
+* blockchain validation;
+* HDWallet support (Icarus, Daedalus or customs);
+* transaction engine: input selection, fee estimation, signing and validation;
+
+This crates is aimed to support as many architectures as possible with as little dependencies as possible.
```

Comment:
This crates package aims to support as many architectures as possible with as little dependencies as possible.